docs/source/advanced/advanced_gpu.rst

```diff
@@ -117,7 +117,7 @@ To activate parameter sharding, you must wrap your model using provided ``wrap``
 When not using Fully Sharded these wrap functions are a no-op. This means once the changes have been made, there is no need to remove the changes for other plugins.
 
 ``auto_wrap`` will recursively wrap `torch.nn.Modules` within the ``LightningModule`` with nested Fully Sharded Wrappers,
-signalling that we'd like to partition these modules across data parallel devices, discarding the full weights when not required (information `here <https://fairscale.readthedocs.io/en/latest/api/nn/fsdp_tips.html>`__).
+signalling that we'd like to partition these modules across data parallel devices, discarding the full weights when not required (information `here <https://fairscale.readthedocs.io/en/latest/api/nn/fsdp.html>`__).
```

Comment:
We could add fairscale to the intersphinx configuration instead to resolve these automatically

Reply:
@ethanwharris I have added `fairscale` and `graphcore` to `intersphinx_mapping`. Not sure how to add reference certain sections of their docs.